airbyte-integrations/connectors/source-railz/source_railz/manifest.yaml

```diff
@@ -0,0 +1,362 @@
+    transformations:
+      - type: AddFields
+        fields:
+          - path: ["businessName"]
+            value: "{{ stream_slice.connection.businessName }}"
+          - path: ["serviceName"]
+            value: "{{ stream_slice.connection.serviceName }}"
+
+  customers_stream:
+    $ref: "*ref(definitions.base_service_stream)"
+    $options:
+      name: "customers"
+      path: "/customers"
+      serviceNames: ["freshbooks", "quickbooks", "quickbooksDesktop", "xero", "oracleNetsuite", "sageBusinessCloud", "sageIntacct", "dynamicsBusinessCentral", "wave", "myob"]
+    primary_key: "id"
+  accounts_stream:
+    $ref: "*ref(definitions.base_service_stream)"
+    $options:
+      name: "accounts"
+      path: "/accounts"
+      serviceNames: ["freshbooks", "quickbooks", "quickbooksDesktop", "xero", "oracleNetsuite", "sageBusinessCloud", "sageIntacct", "dynamicsBusinessCentral", "wave", "myob"]
+    primary_key: "id"
+  inventory_stream:
+    $ref: "*ref(definitions.base_service_stream)"
+    $options:
+      name: "inventory"
+      path: "/inventory"
+      serviceNames: ["freshbooks", "quickbooks", "quickbooksDesktop", "xero", "oracleNetsuite", "sageBusinessCloud", "sageIntacct", "dynamicsBusinessCentral", "wave", "myob"]
+    primary_key: "id"
+  tax_rates_stream:
+    $ref: "*ref(definitions.base_service_stream)"
+    $options:
+      name: "tax_rates"
+      path: "/taxRates"
+      serviceNames: ["freshbooks", "quickbooks", "quickbooksDesktop", "xero", "oracleNetsuite", "sageIntacct", "dynamicsBusinessCentral", "wave", "myob"]
+    primary_key: "id"
+  tracking_categories_stream:
+    $ref: "*ref(definitions.base_service_stream)"
+    $options:
+      name: "tracking_categories"
+      path: "/trackingCategories"
+      serviceNames: ["freshbooks", "quickbooks", "quickbooksDesktop", "xero", "oracleNetsuite", "sageIntacct", "myob"]
+    primary_key: "id"
+  vendors_stream:
+    $ref: "*ref(definitions.base_service_stream)"
+    $options:
+      name: "vendors"
+      path: "/vendors"
+      serviceNames: ["freshbooks", "quickbooks", "quickbooksDesktop", "xero", "oracleNetsuite", "sageBusinessCloud", "sageIntacct", "dynamicsBusinessCentral", "wave", "myob"]
+    primary_key: "id"
+  bank_accounts_stream:
+    $ref: "*ref(definitions.base_service_stream)"
+    $options:
+      name: "bank_accounts"
+      path: "/bankAccounts"
+      serviceNames: ["plaid"]
+    primary_key: "accountId"
+
+# Incremental Service
+  base_incremental_service_stream:
+    $ref: "*ref(definitions.base_service_stream)"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      requester:
+        $ref: "*ref(definitions.requester)"
+        request_options_provider:
+          request_parameters:
+            businessName: "{{ stream_slice.connection.businessName }}"
+            serviceName: "{{ stream_slice.connection.serviceName }}"
+            startDate: "{{ format_datetime(stream_slice.start_time, '%Y-%m-%d') }}"
```

Comment:
why don't we inject this from the stream slicer instead of here?

Reply:
The datetime slicer operates internally with datetime objects
and it's ok for me because my cursor field also datetime.

But API query params have to be a date ( not datetime ) and here I just shorten  datetime -> date